CHANGELOG-1.14.md

```diff
@@ -272,7 +272,7 @@ filename | sha512 hash
     * While this is a backwards-incompatible change, it would have been impossible to setup reliable monitoring around these metrics since the labels were not stable. 
 * Add a configuration field to shorten the timeout of validating/mutating admission webhook call. The timeout value must be between 1 and 30 seconds. Default to 30 seconds when unspecified.  ([#74562](https://github.com/kubernetes/kubernetes/pull/74562), [@roycaihw](https://github.com/roycaihw))
 * client-go: PortForwarder.GetPorts() now contain correct local port if no local port was initially specified when setting up the port forwarder ([#73676](https://github.com/kubernetes/kubernetes/pull/73676), [@martin-helmich](https://github.com/martin-helmich))
-* # Apply resources from a directory containing kustomization.yaml ([#74140](https://github.com/kubernetes/kubernetes/pull/74140), [@Liujingfang1](https://github.com/Liujingfang1))
+* Apply resources from a directory containing kustomization.yaml ([#74140](https://github.com/kubernetes/kubernetes/pull/74140), [@Liujingfang1](https://github.com/Liujingfang1))
     *         kubectl apply -k dir
     *         # Delete resources from a directory containing kustomization.yaml.
```

Comment:
/hold

the whole block needs to be formatted better:
```
        # Apply resources from a directory containing kustomization.yaml
        kubectl apply -k dir

        # Delete resources from a directory containing kustomization.yaml.
        kubectl delete -k dir

        # List resources from a directory containing kustomization.yaml
        kubectl get -k dir
```

i.e. not using bullets.